Week1/homework/2-Error debugging.js

```diff
@@ -0,0 +1,4 @@
+'use strick'
+
+
+console.log("I'm awesome");
```

Comment:
This is one way to solve this issue. But I'm going to give you another challenge. What if we have decided we will only use single quotes for strings...
Can you come up with another solution in that case?

Reply:
in single quotes, we can use \ or ` ` ex : console.log(`I'm awesome`);  //or

console.log('I \'m awesome');